differs/rpm_diff.go

```diff
@@ -123,6 +123,20 @@ func rpmDataFromContainer(image pkgutil.Image) (map[string]util.PackageInfo, err
 
 		defer client.RemoveImage(imageName)
 		defer logrus.Infof("Removing image %s", imageName)
+	} else if image.IsCloud() {
+		// if it's a remote image and not in the daemon, we need to pull it
+		if _, err := client.InspectImage(imageName); err != nil {
+			logrus.Debugf("Pulling cloud image %s", imageName)
+			var buf bytes.Buffer
+			pullOpts := godocker.PullImageOptions{
+				Repository:   image.Source,
+				OutputStream: &buf,
+			}
+			if err := client.PullImage(pullOpts, godocker.AuthConfiguration{}); err != nil {
+				return packages, fmt.Errorf("Error pulling remote image: %s", err)
+			}
```

Comment:
should we do a `logrus.Debug(buf.String())` here? otherwise why pass a buffer to the `OutputStream`?